man/dockerd.8.md

```diff
@@ -404,7 +404,7 @@ Managing the thin-pool outside of Engine makes for the most feature-rich
 method of having Docker utilize device mapper thin provisioning as the
 backing storage for Docker containers. The highlights of the lvm-based
 thin-pool management feature include: automatic or interactive thin-pool
-resize support, dynamically changing thin-pool features, automatic thinp
+resize support, dynamically changing thin-pool features, automatic thin
```

Comment:
Are you sure it's a typo? I think `thinp` is the actual name here (for thin provisioning) https://fedoraproject.org/wiki/Features/ThinProvisioning

/cc @rhvgoyal